packetbeat/etc/fields.yml

```diff
@@ -185,147 +177,169 @@ flows_event:
       description: >
         Second innermost VLAN address used in network packets.
 
-    - name: ip4_source
-      description: >
-        Innermost IPv4 source address as indicated by first packet seen for the
-        current flow.
 
-    - name: ip4_source_location
-      type: geo_point
-      example: "40.715, -74.011"
+    - name: source
+      type: group
       description: >
-        The GeoIP location of the `ip4_source` IP address. The field is a string
-        containing the latitude and longitude separated by a comma.
+        Properties of the source host
+      fields:
+        - name: mac
+          description: >
+            Source MAC address as indicated by first packet seen for the current flow.
 
-    - name: ip4_dest
-      description: >
-        Innermost IPv4 destination address as indicated by first packet seen for the
-        current flow.
+        - name: ip4
+          description: >
+            Innermost IPv4 source address as indicated by first packet seen for the
+            current flow.
```

Comment:
Can you use spaces instead of tabs? Just to keep the same formatting as in the rest of the file.